src/PubNub/PubNub.php

```diff
@@ -89,7 +91,7 @@ public function __construct($initialConfig)
         $this->subscriptionManager = new SubscriptionManager($this);
         $this->telemetryManager = new TelemetryManager();
         $this->tokenManager = new TokenManager();
-        $this->logger = new Logger('PubNub');
+        $this->logger = new \Psr\Log\NullLogger();
```

Comment:
I suggest importing class and therefore:
```suggestion
        $this->logger = new NullLogger();
```